botCommands/club_40_gifs.json

```diff
@@ -63,4 +63,8 @@
     "gif": "https://i.imgur.com/DVrRqoF.gif",
     "author": "Sully"
   }
+  {
+    "gif": "https://media.tenor.com/VRYCq0ZeasQAAAAC/dundie-award.gif",
+    "author": "Unknown"
```

Comment:
But As @zachmmeyer said there's nobody with that name "elrolamenton". So I tagged as "Unknown"

Can I tag that name now?

Reply:
There is nobody with that name in the Discord server, but given that is the tenor username I think we should still tag it? @zachmmeyer thoughts?

Reply:
Requirements have it set to Unknown so anyone in Club-40 who can prove they made it can have their name there.
I posted the original Tenor link in the PR as well.